src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Commands.ScenarioTests.ResourceManager.Common.csproj

```diff
@@ -52,6 +52,9 @@
     <Reference Include="Microsoft.Azure.Management.ResourceManager">
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.ResourceManager.1.2.0-preview\lib\net45\Microsoft.Azure.Management.ResourceManager.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Azure.Management.ServiceFabric">
```

Comment:
removed. 

---
In reply to: [111838002](https://github.com/Azure/azure-powershell/pull/3783#discussion_r111838002) [](ancestors = 111838002)